proto/frequenz/api/common/electrical.proto

```diff
@@ -0,0 +1,199 @@
+// Contains definitions for electrical metrics (AC and DC).
```

Comment:
Maybe name the package `electrical_metrics` to keep consistency of names (the other are nouns and this one is an adjective). Or maybe is it worth to even nest it if we think at some point we could have other metrics definitions: `frequenz.api.common.metrics.electrical`.

Reply:
https://github.com/frequenz-floss/frequenz-api-common/pull/4